### DISCUSSION TITLE
Multiple profiles with single SSO login

### DISCUSSION THREAD

**Discussion by pawana** [Sunbird-RC/community#225] [Ideas & Enhancements]
Hi @dileepbapat @tejash-jl 

Is there a way to create multiple user profiles with single SSO login?

ex.- I can login as a parent and can read and update detailed profiles, create claims of children associated under my account.

cc: @coolbung @parthlawate

**Comment by tejash-jl** [comment]
@pawana 
Yes, you can have multiple logins for a single entity/record.
You can find a sample schema configured with the same.
https://github.com/tejash-jl/edu-core-registries/blob/sample/schemas/Institute.json#L192
Here an institute can have two login/users:
1. Institute user (principal/hod)
2. Institute admin (institute administration office member)
